api/src/Entity/ContentNode.php

```diff
@@ -62,7 +62,7 @@ abstract class ContentNode extends BaseEntity implements BelongsToCampInterface,
      */
     #[ApiProperty(writable: false, example: '/content_nodes/1a2b3c4d')]
     #[Groups(['read'])]
-    public ?ContentNode $root = null;
+    public ContentNode $root;
```

Comment:
Die Property im schema not null zumachen war nicht so einfach?
Oder kommt das noch?

Reply:
Doch, das geht auch - danke für den Input.